action.yml

```diff
@@ -24,6 +24,10 @@ inputs:
     description: 'Include the title in addition to the body in the regex target'
     required: false
     default: "0"
+  sync-labels:
+    description: 'Remove the label from the issue if the label regex does not match'
+    required: false
+    default: "1"
```

Comment:
Arguably, this should be 0 by default. The current behavior does not make sense. This action tries removing labels that do not exist, throwing a bunch of 404 errors.

This would probably require releasing as a major though, so I'll leave it up to you.

Reply:
Agreed. I don't mind doing a major for this. I'll get this updated and merged in

Reply:
IMHO, it would be nice, if this feature ("Remove the label from the issue if the label regex does not match") would still be available.  But of course in the current state it throws a lot of "wrong" errors.  It would be great if you could fix it, but I don't know how much work this is.  Disabling this feature by default is surely better than getting all these errors

Reply:
I should be able to get a release out today or tomorrow which will default to not removing the label but still giving you the option to enable that feature